SampleApps/TestApp/SceneDelegate.swift

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright 2022 Adobe. All rights reserved.
+// This file is licensed to you under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may obtain a copy
+// of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under
+// the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+// OF ANY KIND, either express or implied. See the License for the specific language
+// governing permissions and limitations under the License.
+//
+
+import UIKit
+import SwiftUI
+
+class SceneDelegate: NSObject, UIWindowSceneDelegate {
+    var window: UIWindow?
+    
+    func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
+//        let window = UIWindow(frame: UIScreen.main.bounds)
+//        window.rootViewController = UIHostingController(rootView: ContentView())
+//        self.window = window
+//        window.makeKeyAndVisible()
+        
+        // Use a UIHostingController as window root view controller.
+        if let windowScene = scene as? UIWindowScene {
```

Comment:
let's remove commented code and simplify the if { } with a guard 

guard let windowScene = (scene as? UIWindowScene) else { return }
window = ...

Reply:
Updated to guard statement!